scripts/irods/test/test_federation.py

```diff
@@ -1148,8 +1148,8 @@ def test_catalog_provider_hosts_other_than_the_first_are_considered__issue_6827(
 
 				# If the hostname is not found in the catalog_provider_hosts list, the remote server will sign its local
 				# zone key with its local negotiation key and the signed zone key sent by the local server will not match
-				# that signed zone key. This should cause the ils to result in a SERVER_NEGOTIATION_ERROR.
-				user.assert_icommand(['ils', '-l', remote_home_collection], 'STDERR', 'SERVER_NEGOTIATION_ERROR')
+				# that signed zone key.
+				user.assert_icommand(['ils', '-l', remote_home_collection], 'STDERR', 'SYS_SIGNED_SID_NOT_MATCHED')
```

Comment:
a SID spotting.... can we see if it's 'easy' to use a new error?  that mentions the zone key instead?

Reply:
Created #8378 for that so we can go ahead and get this PR in.